Angular-Newsapp/src/app/short-article/short-article.component.html

```diff
@@ -0,0 +1,32 @@
+<div [className]="'container'">
+    <div [className]="'shortArticle'">
+        <div>
+            <img [className]="'image'" src="../../assets/img/download.png" alt="newsimage">
+        </div>
+        <div [className]="'shortArticleSubRightColumn'">
+            <h4>{{newsHeading}}</h4>
+            <p>{{newsContent| slice:0:500}}...</p>
+            <button id=btnReadMore (click)="readMore(news
+            )">Read More</button>
+            <p></p>
+        </div>
+        <p [className]="'date'">{{date}}</p>
+    </div>
+
+</div>
+<div [id]="news.postId" class="modal">
```

Comment:
the objective is to have the modal in another component and use @input & @output to show the modal